### DISCUSSION TITLE
Display VPN Address at the top or bottom bar

### DISCUSSION THREAD

**Discussion by Canhobix** [haugene/docker-transmission-openvpn#2031] [Ideas]
Hello, 

I would like to suggest an enhancement that could display the VPN status or VPN IP at the Transmission top bar or the status bar.
There is a command that we can run in this docker terminal that we could check the VPN IP and to make sure that we are downloading under the VPN connection.

curl -s checkip.dyndns.org|sed -e 's/.*Current IP Address: //' -e 's/<.*$//'

Thank you

**Comment by pkishino** [comment]
While the idea is good, this would involve modifying whatever gui is being used, that’s not really feasible as we don’t have anything to do with the underlying app nor the UIs..
Also, the container should kill transmission if at any time the vpn connection drops, so that shouldn’t be any problem

**Comment by Canhobix** [comment]
That is right, I thought the same thing, if the VPN connection drops, the transmission container won't start.

Thank you

Cheers
